### PR TITLE
DIV-1170: Fix question requirements

### PR DESCRIPTION
**Pull request by ariannedee**

### CODE REVIEW

**Comment by molund** [comment]
The way no_children is being called, it ends up being  double negative most of the time.  

@no_children(False)

It makes it hard to figure out what it's doing.  Could we make it something like **@if_has_children(True)** instead?

**Comment by ariannedee** [comment]
The argument is the return value if they don't have children. How about:   
  
@if_no_children(return_value=False)
  

  
  

  
  
>

File: edivorce/apps/core/utils/conditional_logic.py [hunk 1]
```diff
@@ -1,33 +1,57 @@
...
+
+@no_children([])
 def get_children(questions_dict):
     children_json = questions_dict.get('claimant_children', '[]')
     if isinstance(children_json, dict):
         children_json = children_json.get('value', '[]')
     return json.loads(children_json)
 
 
+@no_children('0')
```
Comment (molund) [inline edivorce/apps/core/utils/conditional_logic.py:29]: What's the difference between '0' and False?

**Comment by molund** [comment]
That makes more sense.  Disregard my last question about '0'

**Pull request merged**